client/components/formatted-header/index.jsx

```diff
@@ -1,25 +1,44 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
Sorry for the trouble, but could you use JSdoc imports here for the `React` types as well?

Reply:
Why is there a classnames import? :O

I'm on my phone so I can check right now, but we migrated to close recently and there is a lint rule that shouldn't allow using classnames anymore.

Reply:
We haven't migrated Calypso to clsx just yet. The PR is obsolete and I'm planning to redo it soon.

Reply:
Ah, I got.confused about the repo, lol.